bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -1126,6 +1127,23 @@ impl Client {
 
         Ok(())
     }
+
+    /// Checks if a room alias is available in the current homeserver.
```

Comment:
I think we could improve the naming, to something like `alias_exists()` or `alias_is_reserved()`.

When you say available it could mean two things:
1. The alias exists and the room can be resolved using this alias.
2. The alias does not exists and it can be created.

Reply:
That sounds like a good idea as well, though I imagine that this method could then be a common helper method that uses `Client::resolve_room_alias` under the hood and converts the result to a boolean.

Reply:
I added a couple of commits with the changes above in case you want to do another review.